### DISCUSSION TITLE
Multiple user collections in appwrite

### DISCUSSION THREAD

**Discussion by bhavankumar2911** [appwrite/appwrite#5560] [Q&A]
I need to create multiple users' collections programmatically just like any other collections using the server SDK.
Is that even possible?
I hope I'll find some help.
Thank you!

**Comment by joeyouss** [comment]
Hi - What exactly do you need to do? If you mean you need to create collection and assign users to their permissions then you need to give the target user write permissions in the "permissions" field when creating it
https://appwrite.io/docs/server/databases?sdk=nodejs-default#databasesCreateCollection

**Reply by bhavankumar2911** [reply]
Imagine I'm building a B2B application, my users being businesses will have their own user base right? I need to create separate user collection (with login, signup, and other appwrite authentication features) for each of my user. what I'm trying to achieve is kind of [Single-Tenant architecture](https://www.techtarget.com/searchcloudcomputing/definition/single-tenancy#:~:text=Single-tenancy%20is%20an%20architecture%20in%20which%20a%20single,service%20%28SaaS%29%20delivery%20models%20or%20in%20cloud%20services.)
